src/index.js

```diff
@@ -59,35 +71,35 @@ async function setSettings(index) {
   return index.waitTask(taskID);
 }
 
-function infoChange(seq, nbChanges, emoji) {
-  return npm.info().then(npmInfo => {
-    const ratePerSecond = nbChanges / ((Date.now() - loopStart) / 1000);
-    const remaining = ((npmInfo.seq - seq) / ratePerSecond) * 1000 || 0;
-    log.info(
-      `${emoji} Synced %d/%d changes (%d%), current rate: %d changes/s (%s remaining)`,
-      seq,
-      npmInfo.seq,
-      Math.floor((Math.max(seq, 1) / npmInfo.seq) * 100),
-      Math.round(ratePerSecond),
-      ms(remaining)
-    );
-    loopStart = Date.now();
-  });
+async function infoChange(seq, nbChanges, emoji) {
```

Comment:
you  changed the other function to logProcess, should this one also be changed to logChange maybe? although they seem extremely similar. Maybe they can even be merged into a single function

Reply:
They are similar but not quiet, I initially wanted to merge them but I will wait until I split the code and have a better understanding of the whole process.
I'll rename to something similar for consistency 👌